agent/util-scripts/pbench-clear-tools

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
```

Comment:
I thought this is what indent-tabs-mode: t does. The doc says

Indentation can insert tabs if this is non-nil.